go/enclave/db/rawdb/accessors_attestation.go

```diff
@@ -0,0 +1,34 @@
+package rawdb
+
+import (
+	"crypto/ecdsa"
+	"time"
+
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/obscuronet/obscuro-playground/go/common/log"
+
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/ethdb"
+)
+
+func ReadAttestationKey(db ethdb.KeyValueReader, address gethcommon.Address) *ecdsa.PublicKey {
+	key, err := db.Get(attestationPkKey(address))
+	if err != nil {
+		time.Sleep(time.Second)
```

Comment:
Did you mean to leave the sleep and retry in here? Might be good if it at least logs that it's "sleeping and then retrying once" because I could see this causing unforeseen problems if it starts getting hit more than we expect or something.